frontend/packages/console-shared/src/components/dashboard/status-card/AlertItem.tsx

```diff
@@ -82,6 +82,7 @@ const AlertItem: React.FC<AlertItemProps> = ({ alert, documentationLink }) => {
   const alertName = getAlertName(alert);
   const actionObj = getAlertActions(actionExtensions).get(alert.rule.name);
   const { text, action } = actionObj || {};
+  const hideView = alert.hasOwnProperty('hideView') || false;
```

Comment:
IMHO show/hide view belongs to `AlertItemProps` (not to the alert itself) so you can add a new field:
```
export type AlertItemProps = {
  alert: Alert;
  documentationLink?: string;
  hideViewLink?: boolean;
};
```